addons/mail/static/src/messaging/entity/partner/partner.js

```diff
@@ -212,6 +217,13 @@ function PartnerFactory({ Entity }) {
         directPartnerThread: one2one('Thread', {
             inverse: 'directPartner',
         }),
+        composerMentionedPartners: many2one('Composer', {
+            inverse: 'mentionedPartners',
+        }),
+        composerSuggestedMentions: many2one('Composer', {
+            inverse: 'mentionSuggestions',
+
+        }),
```

Comment:
inverse to remove if not needed as explained above, and if needed, there is an unnecessary new line in there